client/src/explorer/model/InstantiatedUnknownTreeItem.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+import { BlockchainTreeItem } from './BlockchainTreeItem';
+import { BlockchainExplorerProvider } from '../BlockchainExplorerProvider';
+import { ChannelTreeItem } from './ChannelTreeItem';
+import * as path from 'path';
+
+export class InstantiatedUnknownTreeItem extends BlockchainTreeItem {
```

Comment:
Can you make an abstract `InstantiatedTreeItem` then make all the other Instantiated tree items extend it please, that way you won't have repeated code in all the tree items

Reply:
We've already got one :) so I've fixed it to extend that